lib/msal-browser/test/cache/BrowserCacheManager.spec.ts

```diff
@@ -2792,56 +2802,50 @@ describe("BrowserCacheManager tests", () => {
             expect(document.cookie).toBe(`${msalCacheKey}=${cacheVal}`);
         });
 
         it("sets samesite", () => {
             const cookieSpy = jest.spyOn(document, "cookie", "set");
-            browserSessionStorage.setItemCookie(msalCacheKey, cacheVal);
+            // @ts-ignore
+            browserSessionStorage.cookieStorage.setItem(msalCacheKey, cacheVal);
             expect(cookieSpy.mock.calls[0][0]).toContain("SameSite=Lax");
         });
 
         it("getItemCookie()", () => {
-            browserSessionStorage.setItemCookie(msalCacheKey, cacheVal);
-            expect(browserSessionStorage.getItemCookie(msalCacheKey)).toBe(
-                cacheVal
-            );
-            expect(browserLocalStorage.getItemCookie(msalCacheKey)).toBe(
-                cacheVal
-            );
-        });
-
-        it("clearMsalCookie()", () => {
-            browserSessionStorage.setItemCookie(msalCacheKey, cacheVal);
-            expect(document.cookie).not.toHaveLength(0);
-            browserSessionStorage.clearMsalCookies();
-            expect(document.cookie).toHaveLength(0);
-
-            const testCookieKey = "cookie";
-            const testCookie = `${testCookieKey}=thisIsACookie`;
-            const testCookieWithPath = "cookie=thisIsACookie;path=/;";
-            browserSessionStorage.setItemCookie(msalCacheKey, cacheVal);
-            expect(document.cookie).not.toHaveLength(0);
-            document.cookie = testCookieWithPath;
-            browserSessionStorage.clearMsalCookies();
-            expect(document.cookie).toBe(testCookie);
-            browserSessionStorage.clearItemCookie(testCookieKey);
+            // @ts-ignore
+            browserSessionStorage.cookieStorage.setItem(msalCacheKey, cacheVal);
+            // @ts-ignore
+            expect(
+                browserSessionStorage.cookieStorage.getItem(msalCacheKey)
+            ).toBe(cacheVal);
+            // @ts-ignore
+            expect(
+                browserLocalStorage.cookieStorage.getItem(msalCacheKey)
+            ).toBe(cacheVal);
         });
 
         it("clearItemCookie()", () => {
-            browserSessionStorage.setItemCookie(msalCacheKey, cacheVal);
-            browserSessionStorage.clearItemCookie(msalCacheKey);
+            // @ts-ignore
+            browserSessionStorage.cookieStorage.setItem(msalCacheKey, cacheVal);
+            // @ts-ignore
+            browserSessionStorage.cookieStorage.removeItem(msalCacheKey);
             expect(document.cookie).toHaveLength(0);
-
-            browserLocalStorage.setItemCookie(msalCacheKey, cacheVal);
-            browserSessionStorage.clearItemCookie(msalCacheKey);
+            // @ts-ignore
+            browserLocalStorage.cookieStorage.setItem(msalCacheKey, cacheVal);
+            // @ts-ignore
+            browserSessionStorage.cookieStorage.removeItem(msalCacheKey);
             expect(document.cookie).toHaveLength(0);
         });
 
+        /* TODO: Move this to new file
```

Comment:
Part of the next PR, or sometime in the unplanned future?

Reply:
Thanks for catching this! Addressed.